s4e-backend/src/main/java/pl/cyfronet/s4e/config/SyncConfig.java

```diff
@@ -0,0 +1,68 @@
+package pl.cyfronet.s4e.config;
+
+import lombok.val;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.task.AsyncTaskExecutor;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+import pl.cyfronet.s4e.admin.sync.PrefixScanner;
+import pl.cyfronet.s4e.admin.sync.task.ChunkedRunner;
+import pl.cyfronet.s4e.admin.sync.task.SyncJobManager;
+import pl.cyfronet.s4e.admin.sync.task.SyncJobStore;
+import pl.cyfronet.s4e.properties.S3Properties;
+import pl.cyfronet.s4e.sync.SceneAcceptor;
+import software.amazon.awssdk.services.s3.S3Client;
+
+import java.time.Clock;
+
+@Configuration
+public class SyncConfig {
+    /// AWS SDK has a default connection pool size of 50, so 40 threads should be fine.
```

Comment:
Is this opinion `, so 40 threads should be fine.` necessary?

Reply:
You're right, it shouldn't be placed here. I'll externalize this param to a `@ConfigurationProperties` class.